### ISSUE TITLE
[WASM] Not receiving websocket messages

### ISSUE THREAD

**Issue by sabin26** [PlugFox/ws#19]
**Describe the bug**
WebSocket message is received on VM & JS environments but not on WASM.

**i.e. webSocketClient.stream.json or webSocketClient.stream.text or webSocketClient.stream.bytes doesn't emit any data when flutter app is compiled to WebAssembly**

**Expected behavior**
WebSocket message is received.

**Desktop (please complete the following information):**

- OS: Windows 10
- Browser Chrome

**Comment by PlugFox** [comment]
Yes, wasm is a pretty fresh compilation target; I'll take a look at that problem in the near future.
